.github/workflows/integration-tests-pr.yml

```
@@ -0,0 +1,67 @@
+on:
+  pull_request:
+  repository_dispatch:
+    types: [acctest-command]
+
+name: Integration tests
+
+jobs:
+  # Maintainer has commented /acctest on a pull request
+  integration-fork:
+    runs-on: ubuntu-latest
+    if:
+      github.event_name == 'repository_dispatch' &&
+      github.event.client_payload.slash_command.sha != '' &&
+      github.event.client_payload.slash_command.tests != '' &&
+      contains(github.event.client_payload.pull_request.head.sha, github.event.client_payload.slash_command.sha)
+
+    steps:
+      - uses: actions/setup-go@37335c7bb261b353407cff977110895fa0b4f7d8 # pin@v2
+        with:
+          go-version: '1.17'
+      - run: go version
+      - uses: actions-ecosystem/action-regex-match@v2
+        id: validate-tests
+        with:
+          text: ${{ github.event.client_payload.slash_command.tests }}
+          regex: '[^a-z0-9_\/]'
+          flags: gi
+
+      # Check out merge commit
+      - name: Checkout PR
+        uses: actions/checkout@v2
+        with:
+          ref: 'refs/pull/${{ github.event.client_payload.pull_request.number }}/merge'
+
+      - run: make TEST_ARGS="${{ github.event.client_payload.slash_command.tests }}" test
```

Could we make this verbose?
```suggestion
      - run: make TEST_ARGS="-v ${{ github.event.client_payload.slash_command.tests }}" test
```